pkg/sidecarproxy/metricshandler/jupyterkernelbusyness/metricshandler.go

```diff
@@ -0,0 +1,162 @@
+package jupyterkernelbusyness
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/v3io/sidecar-proxy/pkg/sidecarproxy/metricshandler"
+	"github.com/v3io/sidecar-proxy/pkg/sidecarproxy/metricshandler/abstract"
+
+	"github.com/nuclio/errors"
+	"github.com/nuclio/logger"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type metricsHandler struct {
+	*abstract.MetricsHandler
+	metric *prometheus.GaugeVec
+}
+
+func NewMetricsHandler(logger logger.Logger,
+	forwardAddress string,
+	listenAddress string,
+	namespace string,
+	serviceName string,
+	instanceName string) (metricshandler.MetricHandler, error) {
+
+	newJupyterKernelBusynessMetricsHandler := metricsHandler{}
+	newAbstractMetricsHandler, err := abstract.NewMetricsHandler(
+		logger.GetChild(string(metricshandler.JupyterKernelBusynessMetricName)),
+		forwardAddress,
+		listenAddress,
+		namespace,
+		serviceName,
+		instanceName,
+		metricshandler.JupyterKernelBusynessMetricName)
+	if err != nil {
+		return nil, errors.Wrap(err, "Failed to create abstract metric handler")
+	}
+
+	newJupyterKernelBusynessMetricsHandler.MetricsHandler = newAbstractMetricsHandler
```

Comment:
Remove `new` prefix from variable names (`jupyterKernelBusynessMetricsHandler`,`abstractMetricsHandler`)
Both confusing with factory functions and redundant (there's no `oldAbstractMetricsHandler` so no need to differentiate)